test/components/Provider.spec.js

```diff
@@ -5,22 +5,28 @@ import PropTypes from 'prop-types'
 import semver from 'semver'
 import { createStore } from 'redux'
 import { Provider, createProvider, connect } from '../../src/index.js'
-import { TestRenderer, enzyme } from '../getTestDeps.js'
+import * as rtl from 'react-testing-library'
```

Comment:
Can we have explicit imports here (`import { render, fireEvent }`)? I tend to dislike `import * as foo` syntax out of both habit and as a very implicit signal that this is TS code (as it's a standard over there, I've noticed. Obviously it doesn't matter at all in the testing context, but it's mainly about enforcing good patterns for others in future PRs. Folks tend to copy what you do in their code.